### PR TITLE
Like count & Date stamp

### PR DESCRIPTION
**Pull request by edchao**
hooked up parse to both of these bad boys.

### CODE REVIEW

**Comment by edchao** [comment]
@donokuda lgty?

File: minigram/GramViewController.swift [hunk 1]
```diff
@@ -119,12 +138,44 @@ class GramViewController: UIViewController {
         // Dispose of any resources that can be recreated.
     }
 
+    // GET PARSE MESSAGE
+    func getGramById(id: String){
+        var query = PFQuery(className:"Message")
+        query.getObjectInBackgroundWithId(id) {
+            (object: PFObject!, error: NSError!) -> Void in
+            if error == nil && object != nil {
```
Comment (donokuda) [inline minigram/GramViewController.swift]: Is there ever a case where `error` will get returned with an object? If not, we can get rid of `error == nil` and just check that `object != nil` and handle the error in the else statement.

File: minigram/HomeViewController.swift [hunk 2]
```diff
@@ -114,11 +117,20 @@ class HomeViewController: UIViewController, UIGestureRecognizerDelegate, UIViewC
         
         // SETUP INIT CONDITIONS
         
+        gramId = ""
         selectedSegue = "gramSegue"
-        getMessages()
+//        getMessages()
```
Comment (donokuda) [inline minigram/HomeViewController.swift]: We can remove this is `getMessages()` works on 127 and 131

**Comment by donokuda** [comment]
Couple of comments. Merge away when you're ready.

**Pull request merged**